src/pages/Embeds/Plio.vue

```diff
@@ -836,8 +907,26 @@ export default {
           if (this.isItemMCQ(itemIndex)) {
             itemResponse.answer = parseInt(itemResponse.answer);
           }
+          if (
+            !this.isItemResponseEmpty(itemIndex, itemResponse.answer) &&
+            !this.isSkipEnabled
+          ) {
+            this.lastAnsweredInteractionIndex = itemIndex;
+          }
```

Comment:
This piece will cause some confusion in one of the edge cases that I've mentioned in the list of comments I've left on the PR. 
This assumes that no unanswered question can exist before an answered question?
Because in the case where there are 4 unanswered questions and then 1 answered question, it will make the last question as `lastAnsweredItemIndex` and the code will never force the user to answer the other questions.

I don't know what the solution should be.
Either we disallow people to change this setting after they publish as it can cause conflict in the responses.
Or we warn them that some older users will not be forced to answer.
Or we update this code to actually force the user to answer all the left out questions in this specific case

Reply:
haan made a suggestion towards the end of the comments. let me know what you think!